internal/apiserver/server.go

```diff
@@ -66,7 +66,7 @@ type apiServer struct {
 func InitConfig() {
 	httpserver.InitHTTPConfig(apiConfig, 5000)
 	httpserver.InitHTTPConfig(spiConfig, 5001)
-	httpserver.InitHTTPConfig(metricsConfig, 6000)
+	httpserver.InitHTTPConfig(metricsConfig, 7000)
```

Comment:
I'm not sure that we want to change the default metrics port in a patch release. Was there a reason for this change?